src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons+Outlined">
+    <title>Shows site app</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo">
+            <a href="#" id="refresh">
+                <img id="imgLogo" src="." alt="movie logo">
+            </a>
+        </div>
+        <nav>
+            <a href="#" class="showCounter">
+                <p>Movies</p>
+                <p id="count"></p>
+            </a>
+            <a href="#">Gender</a>
+            <a href="#">Year</a>
+        </nav>
+    </header>
+    <div class="container" id="container"></div>
+    <div class="" id="modalContainer"></div>
+    <div id="overlay"></div>
```

Comment:
- [ ] As requested by the previous reviewer, it would be nicer to make the modal scrollable so the user is able to scroll and close the modal. As you see in the screenshot below, the modal doesn't fit all the desktop screen sizes. To solve this problem, I would recommend adding a vertical scroll bar to the modal. Try to use dev tools to test your modal on different desktop screen sizes.

<img width="958" alt="image" src="https://user-images.githubusercontent.com/77190203/164604849-c76a666e-d632-42e5-9339-aa3c183ce19a.png">

Reply:
Thank you for the feedback. We have added the scroll option to the modal.
![image](https://user-images.githubusercontent.com/93548973/164609964-711214aa-ae52-4778-8acf-a66842aa1211.png)